widget/widget.go

```diff
@@ -198,25 +199,8 @@ func (w *DisableableWidget) Disabled() bool {
 	return w.disabled
 }
 
-type simpleRenderer struct {
-	content *fyne.Container
-}
-
-func (s *simpleRenderer) Destroy() {
-}
-
-func (s *simpleRenderer) Layout(size fyne.Size) {
-	s.content.Resize(size)
-}
-
-func (s *simpleRenderer) MinSize() fyne.Size {
-	return s.content.MinSize()
-}
-
-func (s *simpleRenderer) Objects() []fyne.CanvasObject {
-	return []fyne.CanvasObject{s.content}
-}
-
-func (s *simpleRenderer) Refresh() {
-	s.content.Refresh()
-}
+// SimpleRenderer is a basic renderer that satisfies widget.Renderer interface by wrapping
+// a single fyne.CanvasObject.
+//
+// Since: 2.1
+type SimpleRenderer = internalWidget.SimpleRenderer
```

Comment:
I think this should be changed to:
```go
func NewSimpleRenderer(object fyne.CanvasObject) *internalWidget.SimpleRenderer {
	return internalWidget.NewSimpleRenderer(object)
}
```
Otherwise, there is no way to use `SimpleRenderer` outside the Fyne repo. 
Also I think that with this change every usage of this in this package (`widget`) should use `NewSimpleRenderer` instead of `internalWidget.NewSimpleRenderer` (to keep consistency).

Reply:
Thanks, I missed the `NewSimpleRenderer`, and I will update the usages.
However the alias should remain, as we should not return `internal` objects from our public API.

Reply:
> Thanks, I missed the NewSimpleRenderer, and I will update the usages.

You're welcome :)

> However the alias should remain, as we should not return internal objects from our public API.

Agree 👍